telemetry/core.schema.json

```diff
@@ -52,7 +52,10 @@
       "type" : "integer"
     },
     "searches" : {
```

Comment:
It doesn't look like we allow an empty object so "minProperties": 1 is advisable